docs/TestingPipelines.md

```diff
@@ -0,0 +1,20 @@
+# Testing overview
+WARP pipelines have accompanying automated tests that run on each Pull Request (PR). These tests compare validated outputs to the new PR outputs. For any changes in code shared between pipelines, the tests also confirm which pipelines could be affected and ensure that the PR makes no unexpected changes to the affected pipelines. 
+
+## WARP branches and how they relate to testing
+
+WARP has three main branches that are used for different stages of pipeline testing: **develop**, **staging**, and **master**: 
+
+| Branch | Purpose | Additional Details |
```

Comment:
@gbggrant @jessicaway is there anything else we'd want to note here about testing on the various branches?